src/test.lisp

```diff
@@ -13,12 +13,6 @@
 
 (declaim (special *suite*))
 
-(defclass test-bundle ()
```

Comment:
Why remove this ?

Reply:
I see two class definitions for `test-bundle`: one in src/test.lisp and one in src/classes.lisp

Aren't these redundant?

Reply:
You're right. Good catch.